src/app/modules/core/advertisement/components/latest/latest-container/latest-container.component.html

```diff
@@ -0,0 +1,17 @@
+<mat-card class="container">
+  <mat-card-content fxLayout="row" fxLayoutAlign="space-between center">
+
+    <button class="arrow-button">
+      <span class="material-icons" (click)="goBehind()">navigate_before</span>
+    </button>
+
+    <cav-latest-mini-card *ngFor="let property of latest | async"
```

Comment:
Se qui cambi il nome come ho scritto dopo, ti viene meno confusionario
`let advertisement of advertisements`